backend/routes/forms/unittesting.py

```diff
@@ -1,25 +1,29 @@
 import base64
-from collections import namedtuple
 from itertools import count
+from pathlib import Path
 from textwrap import indent
+from typing import NamedTuple
 
 import httpx
 from httpx import HTTPStatusError
 
 from backend.constants import SNEKBOX_URL
 from backend.models import Form, FormResponse
 
-with open("resources/unittest_template.py") as file:
+with Path("resources/unittest_template.py").open(encoding="locale") as file:
```

Comment:
We should use `utf8` as the encoding. Not explicitly specifying one to begin with was a mistake,